src/SubjectInfo.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * This file is part of Ares.
+ * Copyright (c) 2015 Václav Vrbka (http://aurielle.cz)
+ */
+
+namespace Grifart\Ares;
+
+use Nette;
+
+
+class SubjectInfo extends Nette\Object
```

Comment:
When I look at it now I see that it is unnecessarily complicated name. Look at them:
- `Ares\SubjectInfo`
- `Ares\Subject`

I think you can't get it wrong from `Ares\Subject` if you know what is Ares about. I'm for `Ares\Subject`.